Xplat/src/main/java/vazkii/botania/common/block/flower/functional/LooniumBlockEntity.java

```diff
@@ -60,7 +62,49 @@ public class LooniumBlockEntity extends FunctionalFlowerBlockEntity {
 			Zombie.class
 	);
 
-	private ResourceLocation lootTable = new ResourceLocation("minecraft", "chests/simple_dungeon");
+	private record StructureLoot(ResourceLocation loc, int weight) {
+		public StructureLoot(ResourceLocation loc) {
+			this(loc, 1);
+		}
+	}
```

Comment:
see below, vanilla has a utility WeightedRandom.Item<T> (or something like that) that is a pair of item and int weight already, no need to remake it.